DOC: Complete API for resample and resample_spec (#10711)
spacetelescope/jwst: docs/jwst/resample/arguments.rst

Reviewer: ```suggestion
    ``x, y`` order. This is the image coordinate to which ``crval`` maps.
```

>         ``output_wcs`` does not have ``bounding_box`` property set.

Reviewer: ```suggestion
        ``output_wcs`` does not have the ``bounding_box`` property set.
```

    measurement time (TMEASURE) found in the image header if available;
    if unavailable, the scaling will be set equal to the exposure time (EFFEXPTM).

``--single`` (boolean, default=False)

>     If set to `True`, resample each input image into a separate output.  If

Reviewer: ```suggestion
    If set to `True`, each image is resampled into a separate output.  If
```
